### ISSUE TITLE
cs_seo prevents staticfilecache from generating static files

### ISSUE THREAD

**Issue by twllndr** [clickstorm/cs_seo#146]
I have cs_seo installed and also tried to use staticfilecache (https://github.com/lochmueller/staticfilecache). In this situation no static files can be generated. I always get this error:

![bildschirmfoto 2017-12-30 um 15 43 03](https://user-images.githubusercontent.com/63435/34457798-b8b5b80e-edbc-11e7-8371-8b479c871886.png)

After deactivating cs_seo, everything works as expected. So does cs_seo set "no_cache = 1" somewhere, which prevents staticfilecache to create the static files? A quick search over the code didn't reveal anything suspicious.

**Comment by mhirdes** [comment]
Hi, thanks for report. I don't use this extension in any project. But we don't set no_cache. Maybe this extension has a problem with user funcs?

**Comment by twllndr** [comment]
Maybe @lochmueller can answer this question or has an idea what the root cause may be?

**Comment by lochmueller** [comment]
Hey @twllndr 
Hey @mhirdes 
Sorry... I don't saw any problems in cs_seo. But also no problems in the core functions used by staticfilecache. Please check your Env and send some more detailed debugging information. Perhaps your TS configuration have some other problems?
Regards,
Tim

**Comment by twllndr** [comment]
Thank you both for checking the issue! I tracked it down to the hreflang generation in cs_seo and found a misconfiguration on my side. I had set the following snippet in my constants file:

```
plugin.tx_csseo {
	hreflang.enable = 1
	hreflang.ids = 0,1
	hreflang.keys = de,en
}
```

But in this project I only had the default language (0/de in my case) configured in TYPO3 - no other languages. After removing the additional language (or deactivating the hreflang generation in general) helped to remove the error. Static files can now be generated without problems.

Sorry I took up your time!
